modules/videoio/misc/java/src/java/videoio+IStreamReader.java

```diff
@@ -0,0 +1,6 @@
+package org.opencv.videoio;
+
+public interface IStreamReader {
+    public long read(byte[] buffer, long size);
```

Comment:
Open question is to keep `size` as a parameter or `buffer.length` is enough (array is initialized). Examples in Java:

https://docs.oracle.com/javase/8/docs/api/java/io/BufferedInputStream.html#read-byte:A-int-int-
https://docs.oracle.com/javase/6/docs/api/java/io/RandomAccessFile.html#read(byte[])
https://docs.oracle.com/javase/6/docs/api/java/io/RandomAccessFile.html#read(byte[],%20int,%20int)